src/frontend/src/icp/derived/icrc.derived.ts

```diff
@@ -105,6 +107,17 @@ export const sortedIcrcTokens: Readable<IcrcCustomToken[]> = derived(
 	([$icrcTokens]) => $icrcTokens.sort(sortIcTokens)
 );
 
+/**
+ * The list of Icrc tokens that are default for Chain Fusion, in the order provided by the static list.
+ */
+export const icrcChainFusionDefaultTokens: Readable<IcrcCustomToken[]> = derived(
+	[icrcTokens],
```

Comment:
`icrcTokens` contains all tokens. For performance reason, could we derive the information from a subset such `icrcDefaultTokens`?

Reply:
I wasn't sure that `icrcDefaultTokens` would contain `ICRC_CHAIN_FUSION_DEFAULT_LEDGER_CANISTER_IDS`

I will change it